<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyOrders</class>
 <widget class="QMainWindow" name="MyOrders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyOrders</string>
  </property>
  <widget class="QWidget" name="orderTabs">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_all">
       <attribute name="title">
        <string>全部</string>
       </attribute>
       <widget class="QTableView" name="table_view_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>581</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="inboxButton">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>收件箱</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_pending">
       <attribute name="title">
        <string>待支付</string>
       </attribute>
       <widget class="QTableView" name="table_view_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>581</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_departure">
       <attribute name="title">
        <string>待出发</string>
       </attribute>
       <widget class="QTableView" name="table_view_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>481</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QListView" name="listView_departure">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>91</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_viewTicket">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>查看登机牌</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancelOrderButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>280</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>取消订单</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_completed">
       <attribute name="title">
        <string>已完成</string>
       </attribute>
       <widget class="QTableView" name="table_view_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>481</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QListView" name="listView_completed">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>70</y>
          <width>91</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_rate">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>评价</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
